From: Etienne Millon <etienne.millon@gmail.com>
Date: Thu, 12 Apr 2012 18:01:45 +0200
Subject: Fix build with gcc 4.7

Headers in the new C++ library do not implicitly import unistd.h, so it is
necessary to do it in the source.

Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=667429
---
 src/tools/depbuild.cpp |    1 +
 1 file changed, 1 insertion(+)

diff --git a/src/tools/depbuild.cpp b/src/tools/depbuild.cpp
index 1e4bbc5..ea0854c 100644
--- a/src/tools/depbuild.cpp
+++ b/src/tools/depbuild.cpp
@@ -25,6 +25,7 @@ This program generates dependencies for all C/C++/Assembly files
 #include <string>
 #include <cstdio>
 #include <cstdlib>
+#include <unistd.h>
 using namespace std;
 
 #include "fileutil.h"
